src/scanner/types.ts

```diff
@@ -1,8 +1,12 @@
+import { ScanResult } from 'snyk-docker-plugin';
+
 import { LegacyPluginResponse } from './images/docker-plugin-shim';
 
 export interface IScanResult {
   image: string;
   imageWithDigest?: string;
   imageWithTag: string;
+  /** @deprecated Will be removed once all customers have safely upgraded to newer versions. */
```

Comment:
Do we have a dashboard for version share?

Reply:
Yes, it's in chartio :)